.github/actions/kots-e2e/action.yml

```diff
@@ -125,6 +150,15 @@ runs:
         aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
         aws-region: ${{ inputs.aws-region }}
 
+    - name: Remove Cluster
+      id: remove-cluster
+      uses: replicatedhq/replicated-actions/remove-cluster@v1
+      if: ${{ inputs.k8s-cluster-skip-teardown != 'true' && (inputs.k8s-distribution == 'k3s' || inputs.k8s-distribution == 'kind' || inputs.k8s-distribution == 'eks') }}
```

Comment:
how about just checking if that outputted `cluster-id` is not empty from the `create-cluster` step instead of checking for individual distributions?

Reply:
nice idea - that output was just added today!